ocs_ci/ocs/tests/test_fiojob.py

```diff
@@ -0,0 +1,179 @@
+# -*- coding: utf8 -*-
+
+from unittest.mock import patch, Mock
+import logging
+import os
+import textwrap
+
+import pytest
+import yaml
+import yaml.parser
```

Comment:
the same question with import like on line :  https://github.com/red-hat-storage/ocs-ci/pull/1714/files#diff-feeffa1c70f9fe781042e2bce426092aR18